ui/raidboss/data/06-ew/dungeon/another_aloalo_island.ts

```diff
@@ -0,0 +1,1550 @@
+      run: (data) => data.ketuHydroCount = 5,
+      outputStrings: {
+        stack: Outputs.stackThenSpread,
+        spread: Outputs.spreadThenStack,
+      },
+    },
+    {
+      id: 'AAI Ketuduke Angry Seas',
+      type: 'StartsUsing',
+      netRegex: { id: '8AC1', source: 'Ketuduke', capture: false },
+      response: Responses.knockback(),
+    },
+    {
+      id: 'AAI Ketuduke Fluke Typhoon Tower',
+      type: 'Ability',
+      netRegex: { id: '8AB0', source: 'Ketuduke', capture: false },
+      suppressSeconds: 5,
+      alertText: (_data, _matches, output) => output.text!(),
+      outputStrings: {
+        text: {
+          en: 'Get Tower',
+          ja: '塔踏み',
+        },
+      },
+    },
+    // ---------------- second trash ----------------
+    {
+      id: 'AAI Wood Golem Ancient Aero III',
+      type: 'StartsUsing',
+      netRegex: { id: '8C4C', source: 'Aloalo Wood Golem' },
+      response: Responses.interruptIfPossible(),
+    },
+    {
+      id: 'AAI Wood Golem Tornado',
+      type: 'StartsUsing',
+      netRegex: { id: '8C4D', source: 'Aloalo Wood Golem' },
+      infoText: (data, matches, output) => {
+        if (data.me === matches.target)
+          return output.itsme!();
+        return output.text!({ player: data.party.member(matches.target) });
+      },
+      outputStrings: {
+        itsme: {
+          en: 'Tornado on YOU',
+          ja: '自分にトルネド',
+        },
+        text: {
+          en: 'Tornado on ${player}',
+          ja: 'トルネド: ${player}',
+        },
+      },
+    },
+    {
+      id: 'AAI Wood Golem Tornado Esuna',
+      type: 'Ability',
+      netRegex: { id: '8C4D', source: 'Aloalo Wood Golem' },
+      condition: (data) => data.role === 'healer' || data.job === 'BRD',
+      alertText: (data, matches, output) =>
+        output.text!({ player: data.party.member(matches.target) }),
+      outputStrings: {
+        text: {
+          en: 'Cleanse ${player}',
+          ja: 'エスナ: ${player}',
+        },
+      },
+    },
+    {
+      id: 'AAI Wood Golem Ovation',
+      type: 'StartsUsing',
+      netRegex: { id: '8BC1', source: 'Aloalo Wood Golem', capture: false },
+      response: Responses.getBehind(),
+    },
+    {
+      id: 'AAI Islekeeper Ancient Quaga',
+      type: 'StartsUsing',
+      netRegex: { id: '8C4E', source: 'Aloalo Islekeeper', capture: false },
+      response: Responses.bleedAoe(),
+    },
+    {
+      id: 'AAI Islekeeper Gravity Force',
+      type: 'StartsUsing',
+      netRegex: { id: '8BC5', source: 'Aloalo Islekeeper' },
+      infoText: (data, matches, output) => {
+        if (data.me === matches.target)
+          return output.itsme!();
+        return output.text!({ player: data.party.member(matches.target) });
+      },
+      outputStrings: {
+        itsme: {
+          en: 'Gravity on YOU',
+          ja: '自分にグラビデフォース',
+        },
+        text: {
+          en: 'Gravity on ${player}',
+          ja: 'グラビデフォース: ${player}',
+        },
+      },
```

Comment:
This could just use the built-in `stackMarkerOn` response from [responses.ts](https://github.com/quisquous/cactbot/blob/main/resources/responses.ts)